src/processRequest.mjs

```diff
@@ -193,6 +197,14 @@ export const processRequest = (
               createError(413, `${maxFiles} max file uploads exceeded.`)
             )
 
+          if (typeof operations !== 'object')
```

Comment:
I'm not sure this check is necessary, because `operations` has already been checked at this point?